distro/pkg/rpm/libyang.spec

```diff
@@ -34,21 +43,44 @@ written (and providing API) in C.
 
 %prep
 %autosetup -p1
-mkdir build
 
 %build
-cd build
-cmake \
-    -DCMAKE_INSTALL_PREFIX:PATH=%{_prefix} \
-    -DCMAKE_BUILD_TYPE:String="Release" \
-    -DCMAKE_C_FLAGS="${RPM_OPT_FLAGS}" \
-    -DCMAKE_CXX_FLAGS="${RPM_OPT_FLAGS}" \
-    ..
-make
+%cmake \
+   -DCMAKE_INSTALL_PREFIX:PATH=%{_prefix} \
+   -DCMAKE_BUILD_TYPE:String="Package" \
```

Comment:
Build type `Package` is obsolete and does not exist. `RELWITHDEBINFO` should be used instead.